chatmm/src/main/java/com/likeminds/chatmm/search/view/LMChatSearchFragment.kt

```diff
@@ -39,6 +43,8 @@ class LMChatSearchFragment : BaseFragment<LmChatFragmentSearchBinding, SearchVie
 
     private lateinit var mAdapter: SearchAdapter
 
+    private var searchExtras: LMChatSearchExtras? = null
```

Comment:
make it as `lateinit var`

Reply:
Nullable as it is not required from all the cases